packages/compass-schema-validation/src/components/validation-editor/validation-editor.jsx

```diff
@@ -93,7 +96,10 @@ class ValidationEditor extends Component {
       textCompleter,
       props.fields
     );
-    this.debounceFetchSampleDocuments = debounce(this.props.fetchSampleDocuments, 750);
+    this.debounceValidatorChanged = debounce((validator, errors) => {
```

Comment:
Nitpick: Isn't errors a boolean signifying whether there are errors? The variable name `errors` makes it sound like it is an array of arrays to me.